core/controllers/question_editor.py

```diff
@@ -61,10 +61,10 @@ def post(self):
                 (question_dict['id'] is not None) or
                 ('question_state_data' not in question_dict) or
                 ('language_code' not in question_dict) or
-                (question_dict['version'] != 1)):
+                (question_dict['version'] != 0)):
```

Comment:
Why are we still dependent on the version number provided from the user to API? I this handler creates a new question then it should initiate its own version number and maybe send the question_id and version as a response?

[Wanted to check the specific reason for API to depend on this value?]